docs/recipes/Deploying-Modules.md

```diff
@@ -0,0 +1,55 @@
+[👈 Return to Overview](./Recipes.md)
+
+# Deploying Modules
+
+## 📖 Table of Contents
+
+* [Deploying to Now with GitHub Actions](#deploying-to-now-with-github-actions)
+
+# Deploying to Now with GitHub Actions
+
+[Now.sh](https://zeit.co/home) allows you to deploy statics such as your modules. We will accomplish this be using [GitHub Actions](https://github.com/features/actions).
+
+## Creating your Deploy Action
+
+This deploy action uses Zeit's `now-cli` which you will need a [token](https://zeit.co/account/tokens) from Now. The below action runs whenever a release is created or a push to master. This will create 
+
+```yml
+name: Deploy
+
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - name: Cache NPM Dependencies
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+    - name: Install Dependencies
+      run: npm install
+      env:
+        NODE_ENV: development
+    - name: Build One App Page Wrapper
+      run: npm run build
+      env:
+        NODE_ENV: production
+    - name: 'Deploy'
+      run: now build --prod --confirm -t $NOW_TOKEN
```

Comment:
The static assets will be published from the build folder, is there a way to output the resulting url that nowsh gives you back after deploying? something along the lines of "successfully deployed assets to `https://subdomain.nows.sh`"

Reply:
So if you watch the action, something like this will appear

```bash
Run now build --prod --confirm -t $NOW_TOKEN
Now CLI 17.1.1
The `name` property in now.json is deprecated (https://zeit.ink/5F)
- Setting up project
Linked to jamesrsingleton/one-app-page-wrapper (created .now and added it to .gitignore)
Inspect: https://zeit.co/jamesrsingleton/one-app-page-wrapper/if0lpn2bd [671ms]
- Building
- Completing
Production: https://one-app-page-wrapper.now.sh [3s]
https://one-app-page-wrapper-if0lpn2bd.now.sh
```

Reply:
So when you create the token you also pick your subdomain?

Reply:
Token and subdomain are separate. The subdomain is generated by Now off of your repo name. You can go into the settings latter to change that if you have a specific subdomain.

Reply:
that makes sense, in surge you can chose it first and then have a file to redeploy to the same domain